bindings/rust/wasmedge-sys/src/lib.rs

```diff
@@ -15,13 +20,23 @@ pub mod vm;
 pub mod wasi;
 
 pub use config::{Config, OptLevel};
+pub use import_obj::ImportObj;
+pub(crate) use io::WasmFnIO;
+pub use io::{I1, I2};
 pub use module::Module;
 pub use raw_result::ErrReport;
 pub use string::{StringBuf, StringRef, WasmEdgeString};
 pub use value::Value;
 pub use version::{full_version, semv_version};
 pub use vm::Vm;
 
+thread_local! {
+    // TODO: allow modify capacity before running
+    #[allow(clippy::type_complexity)]
+    static HOST_FUNCS:
+      RefCell<
+        HashMap<usize, Box<dyn Fn(Vec<Value>) -> Result<Vec<Value>, u8>>>> = RefCell::new(HashMap::with_capacity(var("MAX_HOST_FUNC_LENGTH").map(|s| s.parse::<usize>().expect("MAX_HOST_FUNC_LENGTH should be a number")).unwrap_or(500)));
+}
```

Comment:
So, no need to consider multi-threading here? This is one of the questions I had before making this feature.

Reply:
Currently, multi-threading in WasmEdge is not yet supported. I would like to approve this first. And yes, we are going to implement multi-threading support in near future. So I think we will need to consider in the future PR. Thanks.

Reply:
Let me add an issue first